classes/options-general.php

```diff
@@ -264,17 +264,17 @@ function print_page() {
 
 									<div class="postbox">
 
-										<h3><?php _e( 'Location Defaults', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Location Defaults', 'SimpleMap' ); ?></h3>
 
 										<div class="inside">
-											<p class="sub"><?php _e( 'If most of your locations are in the same area, choose the country and state/province here to make adding new locations easier.', 'simplemap' ); ?></p>
+											<p class="sub"><?php _e( 'If most of your locations are in the same area, choose the country and state/province here to make adding new locations easier.', 'SimpleMap' ); ?></p>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 221 characters

<!-- Comment by [SideCI](https://sideci.com) -->